src/main.py

```diff
@@ -1,5 +1,29 @@
+import textwrap
+
+from mini_games.simple_classics import guess_the_number
```

Comment:
Code from other files can be importet like this, the part after the "from" is the path - directory.file, the part after the "import" is the name of the function